### PR TITLE
Dialog section Code Modular, Break Dialog Section

### PR DESCRIPTION
**Pull request by fragm3**
Fixes https://github.com/fossasia/chat.susi.ai/issues/2114

Changes: Dialog section Code Modular, Break Dialog Section, code refactored

Demo Link: https://pr-2116-1-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd @harsh-jindal please review

File: src/components/Dialog/DialogSection.react.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  tour: { component: <Tour />, size: 'sm' },
+  noComponent: { component: null, size: false },
+};
+
+const DialogSection = props => {
+  const { actions, modalProps, visited } = props;
+
+  const getDialog = () => {
+    if (modalProps && modalProps.isModalOpen) {
+      switch (modalProps.modalType) {
```
Comment (akshatnitd) [inline src/components/Dialog/DialogSection.react.js]: It is equivalent to `DialogData[modalType]`. No switch required.

**Comment by fragm3** [comment]
@akshatnitd made the changes

File: src/components/Dialog/DialogSection.react.js (hunk 1 above)
Comment (fragm3) [reply src/components/Dialog/DialogSection.react.js]: Neat 💯

File: src/redux/reducers/app.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ export default handleActions(
     [actionTypes.APP_SET_VISITED_STATE](state, { payload }) {
       return {
         ...state,
-        visited: true,
+        visited: false,
```
Comment (akshatnitd) [inline src/redux/reducers/app.js]: It should be `true`

Comment (fragm3) [reply src/redux/reducers/app.js]: Missed it while testing tour dialog

**Comment by fragm3** [comment]
@akshatnitd done

**Comment by fragm3** [comment]
@akshatnitd uploading the build

**Pull request merged**